frontend/src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,46 @@
+import { useState } from 'react';
+
+import './Navbar.css';
+import { ReactComponent as Add } from 'assets/icons/add.svg';
+import { ReactComponent as Suggestions } from 'assets/icons/suggestions.svg';
+import { ReactComponent as Home } from 'assets/icons/home.svg';
+import { ReactComponent as Favorites } from 'assets/icons/favorites.svg';
+import { ReactComponent as Messenger } from 'assets/icons/messenger.svg';
+import DropDownMenu from './DropDownMenu';
+
+interface NavbarProps {
+  onCreatePostClick: () => void;
+}
+
+export function Navbar({ onCreatePostClick }: NavbarProps) {
+  const [dropdownMenu, setDropdownMenu] = useState(false);
+
+  return (
+    <div className="navbar flex-box">
```

Comment:
Is div a semantically appropriate element to use for a component that contains some _navigation_ links in the app?